cores/esp32/esp32-hal-rgb-led.h

```diff
@@ -11,8 +11,19 @@ extern "C" {
 #define RGB_BRIGHTNESS 64
 #endif
 
+typedef enum {
+  LED_COLOR_ORDER_RGB,
+  LED_COLOR_ORDER_BGR,
+  LED_COLOR_ORDER_BRG,
+  LED_COLOR_ORDER_RBG,
+  LED_COLOR_ORDER_GBR,
+  LED_COLOR_ORDER_GRB
+} rgb_led_color_order_t;
+
```

Comment:
you can add here:
```cpp
#ifndef LED_COLOR_ORDER_DEFAULT
#define LED_COLOR_ORDER_DEFAULT LED_COLOR_ORDER_GRB
#endif
```

Which would allow to define a different color order in `pins_arduino.h`. You would also need to use `LED_COLOR_ORDER_DEFAULT` in the implementation of `neopixelWrite` above